### DISCUSSION TITLE
OCPP version upgrade

### DISCUSSION THREAD

**Discussion by ChaitanyaYeole02** [villekr/ocpp-asgi#7] [Ideas]
Why is the OCPP version limited to 0.8.3? Is there any reason, or I can use it with higher version too.

**Comment by ChaitanyaYeole02** [comment]
@villekr Also need your view on this?
If you can upgrade the OCPP version, then the library will be able to use new JSONSCHEMA version

**Reply by villekr** [reply]
0.3.0 release has a dependency to currently latest ocpp 0.15.0 version.
